.automatedtesting.json

```diff
@@ -15,7 +15,7 @@
         {
             "NAME": "o3de-extras",
             "URL": "https://github.com/o3de/o3de-extras.git",
-            "BRANCH": "development"
+            "BRANCH": "multi-device-resources"
```

Comment:
The issue is that the CI didn't run through without this. I have already set up https://github.com/o3de/o3de-extras/pull/716 which merges that branch into development and https://github.com/o3de/o3de/pull/18058 to revert this change.

This is the only way I found in order to be able to make these changes since both repositories need the CI to run through in order to merge. Is there another way this can be done?

Also it appears there is an issue with the first one, since the CI already ran but it's not marked as such in the PR… I tried restarting the CI, but it looks like it's not going to work again since the status in the checks is still "Waiting for status to be reported" instead of the usual "This commit is being built" that I see when the CI is running.

Reply:
We can coordinate and bypass AR in a way to merge a o3de-extras and o3de change with some pre-testing, but requires some planning ahead of time. Please work with @o3de/sig-build for future merges like this and we'll be happy to help

Reply:
Ok, we will keep that in mind, thanks :) How exactly does it work then? Do you need to run the AR manually or what's the pre-testing you mentioned?

Reply:
In my opinion, neither this nor our solution is pretty, to be honest. I wrote about o3de-extras in #sig-bulid on discord suggesting a different solution.